src/main/java/discord4j/discordjson/json/GuildStickerModifyRequest.java

```diff
@@ -0,0 +1,20 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableGuildStickerModifyRequest.class)
+@JsonDeserialize(as = ImmutableGuildStickerModifyRequest.class)
+public interface GuildStickerModifyRequest {
+
+    static ImmutableGuildStickerModifyRequest.Builder builder() {
+        return ImmutableGuildStickerModifyRequest.builder();
+    }
+
+    String name();
+    Possible<String> description();
+    String tags();
```

Comment:
All these fields should be `Possible`

https://github.com/Discord4J/Discord4J/pull/1055#issuecomment-1024928888

Reply:
Ready